order-backend/src/main/java/io/eventuate/examples/tram/ordersandcustomers/orders/service/OrderService.java

```diff
@@ -28,6 +28,7 @@
 
   @Transactional
   public Order createOrder(OrderDetailsDTO orderDetails) {
+    entityManager.joinTransaction();
```

Comment:
Why? `entityManager.joinTransaction();`
Doesn't @Transactional take care of transaction management?

Reply:
Sorry, I added it by mistake